src/app/[lang]/admin/login/page.tsx

```diff
@@ -0,0 +1,69 @@
+"use client";
+
+import { Formik } from "formik";
+import * as yup from "yup";
+
+let LoginSchema = yup.object().shape({
+  username: yup.string().required("Please enter your username"),
+  password: yup
+    .string()
+    .required("Password is a required field")
+    .min(8, "Password must be at least 8 characters"),
+});
+
+export default function Login() {
+  return (
+    <div className="min-h-screen flex items-center">
+      <div className="text-center w-full">
+        <h2>Welcome to admin panel</h2>
+        <div className="mt-10">
+          <Formik
+            validationSchema={LoginSchema}
+            initialValues={{ username: "", password: "" }}
+            onSubmit={(values) => {
+              alert(JSON.stringify(values));
+            }}
+          >
+            {({
+              values,
+              errors,
+              touched,
+              handleChange,
+              handleBlur,
+              handleSubmit,
+            }) => (
+              <div className="login">
+                <div className="form">
+                  <form noValidate onSubmit={handleSubmit}>
+                    <input
+                      type="text"
+                      name="username"
+                      onChange={handleChange}
+                      value={values.username}
```

Comment:
You can skip `name`, `onChange`, `onBlur`, `value` props and just use: 

```tsx
<input
  {...getFieldProps("username")}
  placeholder="Enter username"
/>
```

`getFieldProps` can send all these props